test/backend/PlotlyAPI.spec.js

```diff
@@ -66,8 +72,7 @@ describe('Grid API Functions', function () {
                 json.cols[names[5]].data,
                 [130, 140, 150]
             );
-            done();
-        })).catch(done);
```

Comment:
Nice catch! Tthis pattern is everywhere in the tests, I'm not sure how I missed the fact that you can just return a promise. We're on an old-ish version of `mocha` (^2.4.5) but it seems like this was supported since 1.18.0 (https://github.com/mochajs/mocha/blob/master/CHANGELOG.md#1180--2014-03-13, https://github.com/mochajs/mocha/pull/329)